docs/upgrade.html

```diff
@@ -36,7 +36,7 @@
 <ul>
     <li> Java 1.6 is no longer supported. </li>
     <li> Scala 2.9 is no longer supported. </li>
-    <li> Broker IDs above 1000 are now reserved by default to automatically assigned broker IDs. If your cluster has existing broker IDs above that threshold make sure to increase the reserved.broker.max.id broker configuration property accordingly. </li>
+    <li> Broker IDs above 1000 are now reserved by default to automatically assigned broker IDs. If your cluster has existing broker IDs above that threshold make sure to increase the reserved.broker.max.id broker configuration property accordingly. This new feature can be disable by setting broker.id.generation.enable to false. </li>
```

Comment:
I think we need a new section for 0.9.0.1 and / or 0.9.1.0 and leave the description of 0.9.0.0 as is (since it is released and won't change)

Because if we push the doc update now, it will no longer describe the released code. And we do occasionally push doc improvements to the site...

Does that make sense?